src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -267,12 +285,32 @@ export class SentryAppExternalIssueForm extends React.Component<Props, State> {
         dependentField => !hasValue(this.model.getValue(dependentField))
       );
       if (shouldDisable) {
-        field = {...field, disabled: true};
+        fieldToPass = {...fieldToPass, disabled: true};
       }
     }
 
-    //Note that upgrading this to work with the new react select will be quite a challenge!
-    return <FieldFromConfig key={field.name} field={field} {...this.fieldProps(field)} />;
+    //if we have a uri, we need to set extra parameters
+    const extraProps = field.uri
+      ? {
+          loadOptions: (input: string) => this.getOptions(field, input),
+          async: true, //TODO: make configuragble
+          cache: false,
+          onSelectResetsInput: false,
+          onCloseResetsInput: false,
+          onBlurResetsInput: false,
+          autoload: false,
+        }
+      : {};
```

Comment:
Which fields can have `uri`?

Reply:
@billyvg This is only for the select field.